src/App.tsx

```diff
@@ -76,18 +54,14 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {currentrUser && status === 'idle' && posts.length === 0 && (
```

Comment:
make sense to move 'loading', 'failed', 'idle' to status enum as it will make a code cleaner